src/frontend/components/error-boundary/error-boundary.tsx

```diff
@@ -0,0 +1,39 @@
+import { Component } from 'react';
+import ErrorPage from '../../pages/ErrorPage/error-page';
+
+interface ErrorBoundaryProps {}
+
+interface ErrorBoundaryState {
+  hasError: boolean;
+  error: any;
+}
+
+class ErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState> {
+  constructor(props: ErrorBoundaryProps) {
+    super(props);
+    this.state = { hasError: false, error: null };
+  }
+
+  static getDerivedStateFromError(error: Error) {
+    return { hasError: true, error };
+  }
+
+  render() {
+    if (this.state.hasError) {
+      return (
+        <ErrorPage
+          message={
+            'Please send a screenshot of this error to the software team, and try reloading the page to resolve the issue.'
+          }
+          error={this.state.error}
+        >
+          {' '}
+        </ErrorPage>
```

Comment:
you should be able to just use the error page like `<ErrorPage props />` instead of having an empty child